src/wp-content/themes/twentytwentythree/patterns/call-to-action.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Title: Call to action
+ * Slug: twentytwentythree/cta
+ * Categories: featured
+ * Keywords: Call to action
+ * Block Types: core/buttons
+ */
+?>
+<!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide">
```

Comment:
Question on the nesting of tags and the `!<-- /wp:blockname -->` types of comments.

Could we clean this up a bit and be a little more consistent with these? For example, the paragraph block has the opening `<!-- -->` before the `<p>` tag on the line before and the closing one after the closing `</p>` tag on the line after. But then the `button`/`buttons` ones are on the same line as other closing tags.

Personally I am having a hard time reading this because it's not clear and I find my brain wanting to read these like `<!-- Closing site-container -->` type comments that come after closing tags.

If white space does not matter, it would be great to indent with each level as a normal HTML file would be to reflect the nesting of tags/blocks. In some places the nesting is correct (`post-meta.php` pattern is really consistent with this), but in others it's pretty flat (`footer-default.php` is not that great).

Reply:
I agree this markup can be difficult to read! In most cases, this markup has been copied directly from Gutenberg, and the white space does matter in some cases (it can sometimes cause rendering issues in the editor).

I can open an issue in the current TT3 repo to tidy up as much of this markup as possible, where the editor allows.

Reply:
I've created an issue for this here: https://github.com/WordPress/twentytwentythree/issues/179

Reply:
I went ahead and fixed this up in this PR (https://github.com/WordPress/wordpress-develop/pull/3280/commits/4539ce8b539eea1201ae568e42ccf891d0749377).